Specs/Scene/ExpressionSpec.js

```diff
@@ -529,4 +529,110 @@ defineSuite([
         expression = new Expression(new MockStyleEngine(), '(!(1 + 2 > 3)) ? (2 > 1 ? 1 + 1 : 0) : (2 > 1 ? -1 + -1 : 0)');
         expect(expression.evaluate(undefined)).toEqual(2);
     });
+
+    it('evaluates member expression', function() {
```

Comment:
Perhaps name this name "with dot notation" and the test below "with bracket notation."  Likewise, use similar naming in the implementation; "computed" for brackets is confusing.